.gitignore

```diff
@@ -12,3 +12,5 @@ cpu_codegen/
 
 .*.swp
 .nfs*
+build_cmake
+test/**/dump.sh
```

Comment:
What file is this?

Reply:
The file gets generated if you run `model` tester manually so we don't want to accidentally add it to the push.